core/src/lib.rs

```diff
@@ -24,6 +24,29 @@
 
 pub use opendal_core::*;
 
+#[cfg(feature = "services-aliyun-drive")]
+extern crate opendal_service_aliyun_drive;
+#[cfg(feature = "services-azblob")]
+extern crate opendal_service_azblob;
+#[cfg(feature = "services-azdls")]
+extern crate opendal_service_azdls;
+#[cfg(feature = "services-azfile")]
+extern crate opendal_service_azfile;
+#[cfg(feature = "services-fs")]
+extern crate opendal_service_fs;
+#[cfg(feature = "services-ghac")]
+extern crate opendal_service_ghac;
+#[cfg(feature = "services-hdfs-native")]
+extern crate opendal_service_hdfs_native;
+#[cfg(feature = "services-moka")]
+extern crate opendal_service_moka;
+#[cfg(feature = "services-mysql")]
+extern crate opendal_service_mysql;
+#[cfg(feature = "services-s3")]
+extern crate opendal_service_s3;
+#[cfg(feature = "services-vercel-blob")]
+extern crate opendal_service_vercel_blob;
+
```

Comment:
I don't think is the correct fix.

Reply:
`fs` used to be enabled default in opendal but now it's not, so we should add it here

https://github.com/apache/opendal/blob/661e65bb97e56354e7986cdd75b3e01b595d6377/bindings/ocaml/Cargo.toml#L36-L54